.github/workflows/build.yml

```diff
@@ -36,6 +36,12 @@ jobs:
 
       - name: build rook
         run: |
+          # Install kubectl binary as required for generating csv
```

Comment:
In what CI was the mac build failing? I don't see where it's failing.

Reply:
here, https://github.com/rook/rook/actions/runs/4173247229/jobs/7225519549#step:5:749

Reply:
I also think this is random error as it is passing everywhere but good to have fix IMO

Reply:
seems strange that it's intermittent, but good ok if we need to fix it.